src/blueprint/azext_blueprint/_params.py

```diff
@@ -113,7 +113,7 @@ def load_arguments(self, _):
         c.argument('artifact_name', help='A unique name of this resource group artifact.')
         c.argument('display_name', help='Display name of this resource group artifact.')
         c.argument('description', help='Description of the blueprint artifact.')
-        c.argument('depends_on', nargs='+', help='Artifacts which need to be deployed before the specified artifact.')
+        c.argument('depends_on', nargs='*', help='Artifacts which need to be deployed before the specified artifact.')

```

Comment:
May be you can add "To remove depends_on relationships between artifacts, you can use --depends_on, --depends_on= or --depends_on "" in artifact update commands." to help message.

Reply:
updated